Modules/vc-module-notifications/VirtoCommerce.NotificationsModule.Web/Controllers/NotificationsController.cs

```diff
@@ -96,5 +110,142 @@ public ActionResult RenderingTemplate([FromBody]NotificationTemplateRequest requ
 
             return Ok(new { html = result });
         }
+
+        /// <summary>
+        /// Sending test notification
+        /// </summary>
+        /// <remarks>
+        /// Method sending notification, that based on notification template. Template for rendering chosen by type, objectId, objectTypeId, language.
+        /// Parameters for template may be prepared by the method of getTestingParameters. Method returns string. If sending finished with success status
+        /// this string is empty, otherwise string contains error message.
+        /// </remarks>
+        /// <param name="request">Notification request</param>
+        [HttpPost]
+        [Route("template/sendnotification")]
+        public async Task<ActionResult<NotificationSendResult>> SendNotification([FromBody]NotificationRequest request)
```

Comment:
Mark this method by special comment what this API left for backward compatibility and can be replaced  to other. (need to create a new method that allows to send any  notification by API call)

Reply:
mark as [Obsolete]